### DISCUSSION TITLE
[Resolved] "/" causes Penumbra to crash

### DISCUSSION THREAD

**Discussion by SnowdenNamdar** [xivdev/Penumbra#43] [Q&A]
Hi there! I've been using Penumbra for awhile and I'm so thankful for you guys for giving us a TT alternative that won't force use to exit the game to make simple changes.

The issue I'm having right now happened with the current build of Penumbra. I noticed the help mention you can make folders using backslashes, and I thought they were implemented so after I made an empty mod with just a backslash, it crashed Penumbra causing me to restart the launcher.

Then for a second time I tried making a folder with the name of one of the modders whose mods I used the most to test it out and now anytime I try going to my Import Mods it just crashes Penumbra, making me unable to make mod changes on the fly anymore.

The example of what I did was I made an empty mod called "/Rayrei" and its causing the issue I have right now.

Thank you so much for reading my post, I hope I explained everything to the best of my ability for you guys to understand and thank you again for creating an amazing plug in!

**Comment by Ottermandias** [comment]
You shouldn't start with a /, if you want a folder named Rayrei containing Mods, it should look like a path in explorer, i.e. 'Rayrei/', not '/Rayrei'.
But this should not be a problem nor crash you, since empty folders should just be ignored when setting.
But I assume you're using the wrong mechanism (and found a crash completely unrelated to folders, I'm gonna fix that :D )
The Slashes should be in the Sort Order string for a mod, not in its name. On the current version you can edit the sort order either in edit mode or via right-clicking an existing mod.

**Reply by SnowdenNamdar** [reply]
Alright I'll try to do that then! Thank you

**Reply by SnowdenNamdar** [reply]
So I also removed all the mods from the mod folder on my pc and Penumbra works properly again, I also found the folder that kinda broke Penumbra and everythings working fine again!

**Comment by VariableVixen** [comment]
Those aren't backslashes. `/` is a slash or frontslash, `\` is a backslash. Penumbra, to my knowledge, doesn't interpret backslashes specially.
